lib/src/kvs/ds.rs

```diff
@@ -440,7 +469,8 @@ impl Datastore {
 		tx: &mut Transaction,
 		nodes: &[Uuid],
 		this_node_id: &Uuid,
-	) -> Result<Vec<LqValue>, Error> {
+		// TODO this is the issue. We need to track LqValue for stage 2, but also collect errors
```

Comment:
It would be awesome if you could redirect the reader to the definition of "stage 2" or say it in another way so that I can figure out what to do on this later!